src/phylum/ci/ci_github.py

```diff
@@ -175,9 +175,9 @@ def phylum_comment_exists(self) -> bool:
     def repo_url(self) -> Optional[str]:
         """Get the repository URL for reference in Phylum project metadata."""
         # Ref: https://docs.github.com/actions/learn-github-actions/variables#default-environment-variables
-        if server_url := os.getenv("GITHUB_SERVER_URL") is None:
+        if (server_url := os.getenv("GITHUB_SERVER_URL")) is None:
```

Comment:
This is one of the reasons that assignment isn't normally allowed in an `if` statement. I think this is simpler to read and write without the walrus operator

```suggestion
        server_url = os.getenv("GITHUB_SERVER_URL")
        if server_url is None:
```